custom_components/mikrotik_router/mikrotik_controller.py

```diff
@@ -2081,9 +2077,11 @@ def process_kid_control_devices(self):
             ]
         )
 
+        time_diff = self.api.take_client_traffic_snapshot(False)
+
         if not kid_control_devices_data:
             if "kid-control-devices" not in self.notified_flags:
-                _LOGGER.error("No kid-control-devices found on Mikrotik, make sure kid-control feature is configured")
+                _LOGGER.error("No kid control devices found, make sure kid-control feature is configured")
```

Comment:
"On your Mikrotik device" addition there may be good idea. Since this will confuse people and they will try to enable it in integration instead.
Thats happened in the past quite a lot.